feature-group-definitions/fetch-priority.yml

```
@@ -1,4 +1,5 @@
 name: Fetch priority
+description: The optional `priority` parameter of the `fetch()` method, and the `fetchPriority` HTML attribute specifies the priority of the fetch request, relative to other requests of the same type.
```

I know I usually say to put the unique thing first, but I had a really hard time parsing this. What do you think of this?

```suggestion
description: The `fetch()` `priority` option and the `fetchPriorty` HTML attribute give hints to the browser about which requests to do before other requests of the same type.
```